Core/automation/lib/python/core/metadata.py

```diff
@@ -0,0 +1,193 @@
+'''
+Wrapper class for using Item Metadata
+'''
+
+from collections import MutableMapping
+from org.eclipse.smarthome.core.items import Metadata, MetadataKey
+from core.log import logging, LOG_PREFIX
+log = logging.getLogger(LOG_PREFIX + ".metadata")
+from core import osgi
+MetadataRegistry = osgi.get_service("org.eclipse.smarthome.core.items.MetadataRegistry")
+
+__all__ = [ "namespace_exists", "item_metadata", "metadata_namespace" ]
+
+
+def namespace_exists(item_name, namespace):
+    '''Returns True if metadata namespace exists'''
+    return False if MetadataRegistry.get(MetadataKey(namespace, item_name)) is None else True
+
+def _resolveType(value):
+    '''Attempts to resolve the type of the metadata value.
```

Comment:
My preference is to start and end block comments on their own lines too. I think it is easier to read and identify as a comment. WDYT? Although, if we [stuck to the standard](https://www.python.org/dev/peps/pep-0008/#block-comments), we'd be using #s.

Reply:
Like this?
```python
def _resolve_type(value):
    '''
    Attempts to resolve the type of the metadata value.
    '''
````

I have no preference really. I tend to have them on the same line to save on line count. I use VSC so with the highlighting my eyes just ignore quotes when they see green.

Reply:
> Like this?

Yes, that is what I was meaning. Visually, it's more 'blocky', which IMO is easier to read. I use VSC too, but it was tough to read it in GH!

Reply:
That's fair, I hadn't really noticed on GH